graphviz2drawio/models/SvgParser.py

```
@@ -3,11 +3,12 @@
 from collections.abc import Iterable
 from math import isclose
 from xml.etree import ElementTree
+from xml.etree.ElementTree import Element
```

Ruff (0.11.9)

Move standard library import `xml.etree.ElementTree.Element` into a type-checking block

Move into type-checking block

(TC003)